lib/index.js

```diff
@@ -0,0 +1,48 @@
+/* global module, require */
```

Comment:
fixed

On Tue, Jun 7, 2016 at 7:13 PM, tdumitrescu <notifications@github.com>
wrote:

> In lib/index.js
> <https://github.com/tdumitrescu/virtual-jade-loader/pull/2#discussion_r66174830>
> :
>
> > @@ -0,0 +1,48 @@
> > +/* global module, require */
>
> with linter settings for node this should be unnecessary
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tdumitrescu/virtual-jade-loader/pull/2/files/62743da12dc3d8acd7cf50629fbb2c272110231e#r66174830>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAh_UtP5Lb-d3IU2N5OJjWNYiDilkq7Yks5qJgk4gaJpZM4IwWu_>
> .
>